packages/react/src/components/Header/Header.tsx

```diff
@@ -36,7 +37,7 @@ export interface HeaderProps
   description?: ShorthandValue<HeaderDescriptionProps>
 
   /** Align header content. */
-  textAlign?: 'left' | 'center' | 'right' | 'justified'
```

Comment:
My main question is whether we want to have a design term for alignment (like we had here previously) or expose a CSS property like you are doing in this PR.

My opinion is that we should stick with a design term, for that reason `left, center, right, justified` work for me better that all the valid CSS values (`-moz-initial` is definitely not a design term, if you need it, use styles).
And I would even think about replacing `left` and `right` with `start` and `end` to comply with our direction-agnostic terminology.

Reply:
I agree with @miroslavstastny points. I guess @Bugaa92 was just following the way it is already implemented in the `Header` component. Maybe the best option would be to introduce a different prop (something that does not necessarily correlate directly to css prop), and provide the mention values there as options.

Reply:
@miroslavstastny - very good points, I'm confident others agree; I see @mnajdova liked the message so 3 votes already. 

The only issue I have with our `start` and `end` as values of `textAlign` prop is that they mean something different for the actual `textAlign` CSS prop.

So when we'll have

```jsx
<Text textAlign="end" .. />
```
 will we render
```html
<span style="text-align:right" ../>
```
or
```html
<span style="text-align:end" ../>
```
?

These have different functionality in RTL mode as you can see here:
https://codesandbox.io/s/scrollableelement-wm5tb

![Screenshot 2019-07-17 at 16 51 01](https://user-images.githubusercontent.com/5442794/61385662-1b2c2100-a8b3-11e9-805d-fc5e970a452a.png)

The options I see right now are:

1. have `textAlign` prop with values `'start' | 'end' | 'center' | 'justified'` and forward values as they are to CSS `textAlign`


2. have `textAlign` prop with values `'start' | 'end' | 'center' | 'justified'` and:
- forward values as they are to CSS `textAlign` for `center` and `justified`
- convert `start` and `end` prop values to `left` and `right` CSS values

3. keep `textAlign` prop with values `'left' | 'right' | 'center' | 'justified'` (as it is now for `Header`) and forward values as they are to CSS `textAlign`

4. have `textAlign` prop with values `'before' | 'after' | 'center' | 'justified'` and:
- forward values as they are to CSS `textAlign` for `center` and `justified`
- convert `before` and `after` prop values to `left` and `right` CSS values

@mnajdova @miroslavstastny what's your take on all this?

Reply:
FWIW in your codesandbox example you are not using Stardust `Provider`, with that the results are different in RTL:
![image](https://user-images.githubusercontent.com/9615899/61387423-6136b400-a8b6-11e9-9345-fd72fdcf29db.png)

#### Source
```jsx
  <>
  LTR Provider
  <Provider styles={{border: '2px dashed red', padding: '20px'}}>
      <Text textAlign="right">Test align right</Text>
      <Text textAlign="end">Test align end</Text>
  </Provider>
  <br/>
  RTL Provider
  <Provider rtl styles={{border: '2px dashed red', padding: '20px'}}>
      <Text textAlign="right">Test align right</Text>
      <Text textAlign="end">Test align end</Text>
  </Provider>
  </>
```

So this makes `end` CSS value useless in my eyes.

I would go with option 2 and, as @mnajdova suggests, think about renaming textAlign prop to mentally detach it from the CSS prop.

Reply:
> I would go with option 2

done

>  as @mnajdova suggests, think about renaming textAlign prop 

what about having it just `align`? we already have it for other components (`Menu`, `Dropdown`, `FlexItem`) and even though it has different meaning in the other component I think it's obvious what the meaning is for `Text` and `Header`.

So, what do you guys think about renaming `textAlign` prop to `align`? @mnajdova @miroslavstastny

Reply:
Agree, I vote for `align` too

Reply:
done, thanks guys!